libethash-cl/ethash_cl_miner.cpp

```diff
@@ -78,16 +78,30 @@ ethash_cl_miner::~ethash_cl_miner()
 	finish();
 }
 
+// Quite ugly. Saves us a lot of typing on these static functions
+// since we can't keep the platforms vector in the class (static class/functions)
+//
+// LTODO: With a bit of general refactoring this could go away and platforms
+// be queried only once and kept in the class.
+#define ETHASHCL_GET_PLATFORMS(platforms_, failStmt_)	\
```

Comment:
As per our Skype discussion and as per the comment this will get changed very soon to something less ugly